scripts/manage-versions.ts

```diff
@@ -26,10 +28,12 @@ const fs = require('fs');
 const argv = require('yargs').argv;
 const fse = require('fs-extra');
 const generateTreeData = require('./generateData');
+const generateDiffChanges = require('./generateDiffData').default;
```

Comment:
all these requires can be converted to imports right?

Reply:
yup! nice catch https://github.com/icgc-argo/argo-docs/pull/340/commits/b888f4d118c1811063fe9b321b7932a336b0fb71